src/Apps/Auction/Components/Form/Utils/validationSchemas.ts

```diff
@@ -50,3 +50,9 @@ export const biddingValidationSchemas = {
     selectedBid,
   }),
 }
+
+export const editProfileVerificationSchema = Yup.object().shape({
```

Comment:
**suggestion:** What do you think about moving the schema from `src/Apps/Auction/....` to `src/Apps/ColelctorProfile/...`?

Reply:
Ouch why it is in the Auction
Will move to CollectorProfile 👌